fpga_interchange/dedicated_interconnect.cc

```diff
@@ -50,7 +50,7 @@ struct WireNode
 //
 // Routing networks with depth <= kMaxDepth is considers a dedicated
 // interconnect.
-constexpr int kMaxDepth = 20;
+constexpr int kMaxDepth = 4;
```

Comment:
is there any reason why this is being reduced so dramatically? 20 does seem a big high, but 4 also seems low

Reply:
Basically, kMaxDepth was increased also for expansions within a site. It now keeps track only of expansions out of site

Reply:
would increasing it to around 6 still perform decently? that seems more reasonable to be

Reply:
I am not sure on how many routing nodes can be traversed for a dedicated interconnect, where a wire might be divided in multiple nodes. I might as well increase this a bit more though

Reply:
> would increasing it to around 6 still perform decently? that seems more reasonable to be

Sure, I'll test it right away

Reply:
It seems that there is a small but I think negligible hit on performance with 6